CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 * Re-implement `dSYM` copying and stripping to avoid duplicate outputs.  
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#9185](https://github.com/CocoaPods/CocoaPods/issues/9185)  
+* When pod target is a static framework, save time by copying compiled resources  
```

Comment:
needs an extra space here.

Reply:
done!

Reply:
@igor-makarov sorry we released 1.9.2 and I think this needs one last rebase to be in the right spot in the CHANGELOG.

Reply:
@igor-makarov nit this needs one extra line

Reply:
@dnkoutso Done